lib/galaxy/managers/genomes.py

```diff
@@ -86,15 +90,21 @@ def _create_genome_filter(model_class=None):
             # Doesn't filter genome_build for collections
             if model_class.__name__ == "HistoryDatasetCollectionAssociation":
                 return False
-            column = func.json_extract(model_class.table.c._metadata, "$.dbkey")
+            # TODO: should use is_postgres(self.database_connection) in 23.2
```

Comment:
+1. Can you, please, make sure this is done after this is merged into 23.2?

Reply:
Yes, I will make sure of that